core/src/components/datetime/test/locale/datetime.e2e.ts

```diff
@@ -140,60 +132,47 @@ test.describe('ar-EG', () => {
 class DatetimeLocaleFixture {
   readonly page: E2EPage;
   locale = 'en-US';
+  datetime!: Locator;
 
   constructor(page: E2EPage) {
     this.page = page;
   }
 
-  async goto() {
-    await this.page.goto(`/src/components/datetime/test/locale`);
-  }
-
-  async setLocale(locale: string) {
+  async goto(locale = 'en-US', presentation = 'date') {
     this.locale = locale;
-    await this.page.locator('select').selectOption(locale);
-    await this.page.waitForChanges();
+    await this.page.setContent(`
+      <ion-datetime
+        show-default-title="true"
+        show-default-buttons="true"
+        locale="${this.locale}"
+        presentation="${presentation}"
+        value="2022-04-19T04:20:00"
+      ></ion-datetime>
+    `);
+
+    this.datetime = this.page.locator('ion-datetime');
+
+    await this.page.waitForSelector('.datetime-ready');
   }
 
   async expectLocalizedDatePicker() {
```

Comment:
Optional: Now that this, `expectLocalizedMonthYearPicker`, and `expectLocalizedTimePicker` have had most of their behavior stripped, the only difference between them is the name of the screenshot. So they could be merged or stripped entirely.